packages/client/components/ShareTopicRoot.tsx

```diff
@@ -14,26 +13,23 @@ const ShareTopicRoot = () => {
   const location = useLocation<{backgroundLocation?: Location}>()
   const history = useHistory()
 
+  const [open, setOpen] = React.useState(false)
+
   const onClose = useCallback(() => {
     const state = location.state
     history.replace(state?.backgroundLocation ?? `/new-summary/${meetingId}`)
   }, [location])
 
-  const {openPortal, closePortal, modalPortal} = useModal({
-    id: 'shareTopicModal',
-    onClose
-  })
-
   useEffect(() => {
-    openPortal()
+    setOpen(true)
```

Comment:
+1 if I understand correctly, it opens the modal after initial render, why not use `true` by default[here](https://github.com/ParabolInc/parabol/pull/8450/files#diff-803c8f379080453c0db3ef534d257a76a3028246ee6cf1c14602cea5328871eaR16) instead?